plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/src/com/google/cloud/tools/eclipse/appengine/deploy/flex/FlexWarStagingDelegateTest.java

```diff
@@ -56,6 +56,13 @@ public void testStage() {
     IStatus status = delegate.stage(stagingDirectory, safeWorkDirectory,
         null, null, new NullProgressMonitor());
 
+    // Diagnostic message: https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/3107
+    if (!status.isOK()) {
+      System.out.println(status.getSeverity() + ": " + status.getMessage());
+      if (status.getException() != null) {
+        status.getException().printStackTrace(System.out);
+      }
+    }
```

Comment:
Do we want to fail?  I hit this trying out [Photon RC1](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/tree/photon) and used `assertTrue("staging failed: " + status, status.isOK())`

Reply:
We have `assertTrue(status.isOK())` below.

Reply:
Shouldn't we check it first?

Reply:
Done.